iOSDemo/DemoViewController.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import UIKit
+import SweetUIKit
+
+enum DemoItem: Int {
+    case table, edit, collection
+    
+    static var titles: [String] {
+        let allItems: [DemoItem] = [.table, .edit, .collection]
+        return allItems.map{ $0.title }
+    }
+    
+    var title: String {
+        switch self {
+        case .table: return "Table Controller"
+        case .edit: return "Edit Controller"
+        case .collection: return "Collection Controller"
+        }
+    }
+    
+    var viewController: UIViewController {
+        switch self {
+        case .table: return TableController()
+        case .edit: return EditViewController()
+        case .collection: return CollectionController()
+        }
+    }
+}
+
+class DemoViewController: SweetTableController, UITableViewDataSource {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(TableViewCell.self)
+        tableView.dataSource = self
+        tableView.delegate = self
+    }
+    
+    //MARK: - UITableViewDataSource
```

Comment:
Can you move the `UITableViewDataSource` protocol implementation to an extension? We tend to do that.